pkg/amphora/client.go

```diff
@@ -55,12 +76,48 @@ type Client struct {
 	HTTPClient http.Client
 }
 
-const secretShareURI = "/intra-vcp/secret-shares"
+
+const secretShareURI = "/secret-shares"
+
+//GetObjectList query secretIds based on the filter argument (tags)
+func (c *Client) GetObjectList(objectListRequestParams *ObjectListRequestParams) (MetadataPage, error) {
+	var result MetadataPage
+
+	urlParams := url.Values{}
+	urlParams.Add("filter", objectListRequestParams.Filter)
+	urlParams.Add("pageNumber", fmt.Sprintf("%v", objectListRequestParams.PageNumber))
+	urlParams.Add("pageSize", fmt.Sprintf("%v", objectListRequestParams.PageSize))
+	urlParams.Add("sortProperty", objectListRequestParams.SortProperty)
+	urlParams.Add("sortDirection", objectListRequestParams.SortDirection)
```

Comment:
These need to be replaced.
```suggestion
	urlParams.Add("sort", objectListRequestParams.SortProperty)
	urlParams.Add("dir", objectListRequestParams.SortDirection)
```

See 
https://github.com/carbynestack/amphora/blob/master/amphora-common/src/main/java/io/carbynestack/amphora/common/rest/AmphoraRestApiEndpoints.java#L59-L69